account/account.go

```diff
@@ -72,8 +72,9 @@ func (acc *Account) UnmarshalJSON(bs []byte) error {
 }
 
 func (acc Account) Fingerprint() string {
-	return fmt.Sprintf("{ %s %v}",
+	return fmt.Sprintf("{ %s %d %v}",
```

Comment:
Could you please update Fingerprint for validator as well?

Reply:
sure why not